### PR TITLE
Fixes #1171 improvements in readme

### LINKED ISSUE #1171: Minor improvements in README.md
**I'm submitting a ...**
- [ ] bug report
- [x] feature request

**Current behavior:**

**Expected behavior:**

Minor improvements in readme.md. I've filed a PR for the same. Please review.

### PR DESCRIPTION
**Pull request by prabhupant**
Fixes #1171 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [x] I have added necessary documentation (if appropriate)
- [ ] Added Surge preview link

#### Changes proposed in this pull request:

- improvements in readme.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,22 +55,22 @@ $ npm install -g @angular/cli@latest
...
-* **Step 5:** Go to localhost:4200 where the application will be running locally.
+* **Step 5:** Go to [localhost:4200](http://localhost:4200) where the application will be running locally.
 
 ### For deploying with [Github Pages](https://pages.github.com/):
 With these very simple steps you can have susper deployed:
 
 * **Step 1:** Fork susper repository and clone it to your desktop.
 * **Step 2:** Then checkout to your master branch `$ git checkout master`
-* **Step 3:** Deploy running this: ```ng build``` and then ```npm run deploy```
+* **Step 3:** Deploy running this: ```$ ng build``` and then ```$ npm run deploy```
```
Comment (harshit98) [inline README.md]: No need to have `$`. Simple `ng build` and `npm run deploy` commands can be understood.

File: README.md [hunk 2]
```diff
@@ -55,22 +55,22 @@ $ npm install -g @angular/cli@latest
...
 * **Step 4:** Visit `https://yourusername.github.io/susper` and you should see the Susper search running.
-* **Step 5:** As you search you might see that it cant find anything, to resolve this, on search you will see there is a red shield on the search bar, click on it and allow to load unsafe scripts.
+* **Step 5:** As you search you might see that it can't find anything, to resolve this, on search you will see there is a red shield on the search bar, click on it and allow to load unsafe scripts.
 * **Step 6:** Reload and you will have a fully functioning Susper search deployed on GitHub pages.
 
 ### For deploying with [Surge](https://surge.sh/):
 
-Surge will automatically generate deployment link whenever a pull request passes 
-CI. 
+Surge will automatically generate the deployment link whenever a pull request passes
```
Comment (harshit98) [inline README.md]: Change `the` --> `a`.

**Comment by prabhupant** [comment]
@harshit98 did the changes...please review

**Comment by prabhupant** [comment]
@praveenojha33 removed the $ signs. please review

**Pull request merged**